docs/hub/billing.md

```diff
@@ -10,7 +10,7 @@ We offer advanced security and compliance features for organizations through our
 
 The Enterprise Hub is billed like a typical subscription. It renews automatically, but you can choose to cancel it at any time in the organization's billing settings.
 
-You can pay for the Enterprise Hub subscription with a credit card or your AWS account.
+You can pay for the Enterprise Hub subscription with a credit card, with a contract (for more than 50 seats), or your AWS account.
```

Comment:
Let's clarify how to ask to for contract, if that's not already stated elsewhere in the docs?

Reply:
pointing to the flow inside the product would be the best^

Reply:
@VioletteLepercq let me know if you need help for this